docs/Advanced-Usage/FireAxe-Partitioning-onto-Multiple-FPGAs/FireAxe-Overview.rst

```diff
@@ -0,0 +1,73 @@
+FireAxe Overview
+=============================================
+
+Since the initial release of FireSim, the proliferation of open hardware IP has
+enabled researchers to generate new SoC configurations that no longer fit in a
+single FPGA. FireAxe builds on top of FireSim to enable *partitioning of large
+designs onto multiple FPGAs*, overcoming the single-FPGA limitation for monolithic RTL designs.
+
+Partition Modes
+==================
+FireSim uses host-decoupling to control the advancement of the simulator, and
+this causes weird things to happen when the design is partitioned across multiple FPGAs.
+FireAxe provides users with three options (modes) to perform partitioning : exact-mode, fast-mode and the NoC-partition-mode.
+These options are passed on to the compiler which will generate the correct circuitry
+to deal with the odd things that happen on the partition interface.
+A more detailed explanation about the circuitry and the partition modes are provided in
+`the FireAxe paper <https://joonho3020.github.io/assets/ISCA2024-FireAxe.pdf>`_.
+However, if you just want to use FireAxe, the below explanations are sufficient
+for you to get started.
+
+Exact-Mode
+-----------
+In the exact-mode, users can choose the modules to partition out and place on separate FPGAs.
+The partitioned simulation will behave *exactly* the same as when running the target on a software RTL simulator.
+This is usesful when the partition boundary is not latency insensitive (i.e. the interface is not ready-valid interface nor credit based)
+and when the boundary contains combinational logic running through it.
+
+Example modules that can be partitioned out are RoCC accelerators as the ports to access the
+page-table-walker boundaries are combinationally dependent to each other.
+However, when running baremetal tests, address translations are unnecessary
+and users can use the fast-mode to partition out RoCC accelerators.
```

Comment:
```suggestion
An example of a module that can be partitioned out using the exact-mode is a RoCC accelerator since the ports to access the
page-table-walker boundaries are combinationally dependent to each other.
```